packages/xo-web/src/xo-app/home/index.js

```diff
@@ -763,6 +779,20 @@ export default class Home extends Component {
       {label}
     </MenuItem>
   ))
+
+  _updateSelectedPowerStates = powerStates =>
+    this._setFilter(
+      ComplexMatcher.setPropertyClause(
+        this._getParsedFilter(),
+        'power_state',
+        powerStates === ''
+          ? undefined
+          : new ComplexMatcher.Or(
+              powerStates.split(',').map(_ => new ComplexMatcher.String(_))
```

Comment:
As discussed, revert to not using `simpleValue` to avoid having to split the value.